### DISCUSSION TITLE
Bokehpivot widgets not active

### DISCUSSION THREAD

**Discussion by wshobe** [NREL/ReEDS-2.0#173] [General]
When I run the Bokeh server and enter the address of the runs folder, many (most) of the widgets appear as text but are not active. Only the Result and Preset dropdowns work. This is true with Firefox and Chrome on both Mac and Linux. I am using the latest ReEDS-2.0. Any advice on how to get the Bokeh widgets to work?

<img width="456" alt="Screenshot 2024-03-21 at 8 31 41 PM" src="https://github.com/NREL/ReEDS-2.0/assets/6907373/61a44990-a1f0-4b0a-b593-ad4c0cad00e1">

**Comment by kennedy-mindermann** [comment]
Currently, bokeh doesn't work properly when using the 'reeds2' environment. It only works with the old 'reeds' environment. 

If you don't already have that installed, you'll want to check out a previous version of the model and run the following command to create the environment: `conda env create -f environment.yml`

I would recommend checking out the version that was used for the [2023 Standard Scenarios](https://github.com/NREL/ReEDS-2.0/tree/v2023) as that still used the 'reeds' environment.
